features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/semantics/SliderSemanticsNodeMapper.kt

```diff
@@ -0,0 +1,96 @@
+    private val semanticsUtils: SemanticsUtils = SemanticsUtils()
+) : AbstractSemanticsNodeMapper(
+    colorStringFormatter,
+    semanticsUtils
+) {
+    override fun map(
+        semanticsNode: SemanticsNode,
+        parentContext: UiContext,
+        asyncJobStatusCallback: AsyncJobStatusCallback
+    ): SemanticsWireframe {
+        var index = 0
+        val trackWireframe = resolveTrackWireframe(parentContext, semanticsNode, index++)
+        val thumbWireframe = resolveThumbWireframe(parentContext, semanticsNode, index)
+        return SemanticsWireframe(
+            listOfNotNull(trackWireframe, thumbWireframe),
+            null
+        )
+    }
+
+    private fun resolveThumbWireframe(
+        parentContext: UiContext,
+        semanticsNode: SemanticsNode,
+        index: Int
+    ): MobileSegment.Wireframe? {
+        val globalBounds = resolveBounds(semanticsNode)
+        val progressBarRangeInfo = semanticsUtils.getProgressBarRangeInfo(semanticsNode)
+        val progress = progressBarRangeInfo?.let {
+            it.current / (it.range.endInclusive - it.range.start)
+        }
+        val thumbHeight = DEFAULT_THUMB_RADIUS.value * 2 * parentContext.density
+        val yOffset = (globalBounds.height - thumbHeight).toLong() / 2
+        println("prod thumb - height: ${globalBounds.height}, thumbHeight $thumbHeight")
+        println("test thumb - yOffset: $yOffset")
+        return progress?.let {
+            val xOffset = progress * globalBounds.width + globalBounds.x
+            MobileSegment.Wireframe.ShapeWireframe(
+                id = resolveId(semanticsNode, index),
+                x = xOffset.toLong(),
+                y = globalBounds.y + yOffset,
+                width = thumbHeight.toLong(),
+                height = thumbHeight.toLong(),
+                shapeStyle = MobileSegment.ShapeStyle(
+                    backgroundColor = DEFAULT_COLOR,
+                    cornerRadius = thumbHeight / 2
+                )
+            )
+        }
+    }
+
+    private fun resolveTrackWireframe(
+        parentContext: UiContext,
+        semanticsNode: SemanticsNode,
+        index: Int
+    ): MobileSegment.Wireframe {
+        val globalBounds = resolveBounds(semanticsNode)
+        val trackHeight = DEFAULT_TRACK_HEIGHT.value * parentContext.density
+        val yOffset = (globalBounds.height - trackHeight).toLong() / 2
+        return MobileSegment.Wireframe.ShapeWireframe(
+            id = resolveId(semanticsNode, index),
+            x = globalBounds.x,
+            y = globalBounds.y + yOffset,
+            width = globalBounds.width,
+            height = trackHeight.toLong(),
+            shapeStyle = MobileSegment.ShapeStyle(
+                backgroundColor = DEFAULT_COLOR,
+                cornerRadius = trackHeight / 2
+            )
+        )
+    }
+
+    companion object {
+        // TODO RUM-7467: Use contrast color of parent color
+        private const val DEFAULT_COLOR = "#000000FF"
```

Comment:
> [!TIP]
> Probably it's better to put color name into the constant?

Reply:
It will be handled by the ticket mentioned in the comment